BTCPayServer/Payments/Bitcoin/NBXplorerListener.cs

```diff
@@ -330,11 +341,9 @@ private async Task<int> FindPaymentViaPolling(BTCPayWallet wallet, BTCPayNetwork
                     var transaction = await wallet.GetTransactionAsync(coin.OutPoint.Hash);
 
                     var address = network.NBXplorerNetwork.CreateAddress(strategy, coin.KeyPath, coin.ScriptPubKey);
-                    var payJoinSelfContributedAmount =
```

Comment:
There is no payjoin logic in `FindPaymentViaPolling` at all?

Reply:
No, all logic is in the update payment state.